packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -51,6 +51,9 @@ import { createHmac } from 'crypto';
 import { fromBuffer } from 'file-type';
 import { lookup } from 'mime-types';
 
+const requestPromiseWithDefaults:requestPromise.RequestPromiseAPI = requestPromise.defaults({
+	timeout: 300000, // 5 minutes
+});
```

Comment:
Should this be explicitly typed? The `request-promise-native` library uses the [`RequestAPI`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b5bfba2994c91a099cd5bcfd984f6c4c39228e5/types/request-promise-native/index.d.ts#L24) interface from the `request` library, which contains a typed [`defaults()`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b5bfba2994c91a099cd5bcfd984f6c4c39228e5/types/request/index.d.ts#L32).

Reply:
Thanks for pointing that out @ivov 
Just pushed an update without the explicit typing.